lib/factory/EntryFieldDescription.js

```diff
@@ -77,3 +132,7 @@ function escapeText(text) {
 
   return escaped.join('');
 }
+
+function translate(string) {
```

Comment:
What's the purpose of that?

Reply:
Lol, sorry, I forgot to fix that. The issue is that `EntryFieldDescription` now depends on `translate` if we want to translate _More_ and _Less_.